app/lib/reducers/ui.js

```diff
@@ -111,7 +111,7 @@ const reducer = (state = initial, action) => {
         }
 
         if (null != config.colors) {
-          if (state.colors.toString() !== config.colors.toString()) {
+          if (JSON.stringify(state.colors) !== JSON.stringify(config.colors)) {
```

Comment:
It's actually not backwards compatibility since an array is still valid if the 6 x 6 cubes and grey scale need to be specified 😉 . Either way, I added some code to handle an array vs. an object. Let me know if I missed anything!